### PR TITLE
createTables.sql

### PR DESCRIPTION
**Pull request by rioot123**

### CODE REVIEW

File: sql/createTables.sql [hunk 1]
```diff
@@ -3,12 +3,13 @@ CREATE DATABASE ParkingProject;
...
+    UserID INT AUTO_INCREMENT,
     FullName VARCHAR(50) NOT NULL ,
     Email VARCHAR(50) ,
     Password VARCHAR(128) NOT NULL ,
     PhoneNumber VARCHAR(20) NOT NULL ,
-    Manager BOOLEAN NOT NULL DEFAULT FALSE ,
-    PRIMARY KEY (Email)
+    Admin BOOLEAN NOT NULL DEFAULT FALSE ,
+    PRIMARY KEY (UserID)
 ) ENGINE = InnoDB;
```
Comment (weihanlin) [inline sql/createTables.sql]: How does a user can login by? 
UserID and Password? but you make UserID as auto increased integer

File: sql/createTables.sql [hunk 2]
```diff
@@ -19,7 +20,7 @@ CREATE TABLE Location (
...
 
 CREATE TABLE Space (
-    SpaceID INT,
+    SpaceID INT AUTO_INCREMENT,
```
Comment (weihanlin) [inline sql/createTables.sql]: This field doesn't need to be auto_increment

**Review by weihanlin — CHANGES_REQUESTED** [review]
Please alter your change

File: sql/createTables.sql (hunk 1 above)
Comment (rioot123) [reply sql/createTables.sql]: UserID is just 1,2,3,4,5 also i thought we were logging in with email/password?

File: sql/createTables.sql (hunk 2 above)
Comment (rioot123) [reply sql/createTables.sql]: changed

File: sql/createTables.sql (hunk 1 above)
Comment (weihanlin) [reply sql/createTables.sql]: If you'd like user login with email but email isn't primary key, you should add the "NOT NULL" restriction

**Pull request merged**